TrailGage/app/src/main/java/com/example/trailgage/backend/RetrofitClient.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.trailgage.backend
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClient {
+    private const val BASE_URL = "http://trailgage.local/"
```

Comment:
Android won't be able to resolve `.local` addresses by default.

Reply:
yes. i am aware of that. but when i tried to import and instantiate the class i could not do it because there is the exception first and then the the class that i should use.